### PR TITLE
Add linters and yarn

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
> Why are you changing the html identation? Is this suggested by eslint somehow?

To follow our general style guide for html using 2 spaces for indentation.

File: coss/base/templates/base.html [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {% load static wagtailuserbar %}
-
+<!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
```
Comment (comzeradd) [inline coss/base/templates/base.html]: You mean the utf part? It's there, just a few lines below.

**Comment by comzeradd** [comment]
PR updated

**Comment by comzeradd** [comment]
PR rebased

**Comment by akatsoulas** [comment]
@comzeradd I will get to this as soon as we reduce our backlog and deliver the main functionality for the phase 1 launch.

**Pull request closed without merging**